common/script/fns/cron.js

```diff
@@ -36,6 +36,9 @@ module.exports = function(user, options) {
   }
   user.auth.timestamps.loggedin = new Date();
   user.lastCron = now;
+  if (_.isFinite(+user.preferences.timezoneOffset)) {
```

Comment:
`if (timezoneOffset in user.preferences)` won't work correctly. Because this isn't a normal object but a mongoose one, that property will always be present even if undefined